backend/src/email/models/email-message-tx.ts

```diff
@@ -0,0 +1,112 @@
+import {
+  Column,
+  DataType,
+  ForeignKey,
+  Model,
+  Table,
+} from 'sequelize-typescript'
+import { User } from '@core/models'
+import { MessageStatus } from '@core/constants'
+
+// identical to CampaignS3ObjectInterface; to tweak and abstract if necessary
+interface AttachmentS3ObjectInterface {
+  key?: string
+  bucket?: string
+  filename?: string
+  temp_filename?: string
+  error?: string
+}
+
+export enum TransactionalEmailMessageStatus {
+  Unsent = 'UNSENT',
+  // AttachmentTooLargeError = 'ATTACHMENT_TOO_LARGE_ERROR', // unused
+  // TooManyAttachmentsError = 'TOO_MANY_ATTACHMENTS_ERROR', // unused
+  // InvalidParamsError = 'INVALID_PARAMS_ERROR', // unused; these are errors caught by Joi
```

Comment:
listed status in chronological order of appearance

have erred on the side of having as many statuses as possible (realistically we probably don't need so many?)